device/samples/edge_downstream_device.js

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+'use strict';
+
+var fs = require('fs');
+var Protocol = require('azure-iot-device-mqtt').Mqtt;
+// Uncomment one of these transports and then change it in fromConnectionString to test other transports
+// var Protocol = require('azure-iot-device-amqp').AmqpWs;
+// var Protocol = require('azure-iot-device-http').Http;
```

Comment:
I would remove the Http, AmqpWs, and MqttWs lines.  Even though they're commented out, they're invalid for an edge downstream device so we shouldn't imply that they could be used.

Reply:
Makes sense, will remove and update this PR.